rioting-daisies/BangazonWorkforce
# Changed Names and headers on all pages

# Description

Just changed headers and names on all of the pages



Fixes Issue # (link issue ticket here)

## Type of change

Please delete options that are not relevant.




# Testing Instructions
Git fetch --all then checkout into cm-headers
Run bangazon and check that all pages and headers look correct.



# Checklist:
